app/_components/TextExpander.tsx

```
@@ -0,0 +1,34 @@
+	const [isExpanded, setIsExpanded] = useState(false);
+	const displayText = isExpanded
+		? children
+		: children.split(" ").slice(0, 40).join(" ") + "...";
+
+	return (
+		<span>
+			{displayText}{" "}
+			<button
+				className="border-b border-primary-700 pb-1 leading-3 text-primary-700"
+				onClick={() => setIsExpanded(!isExpanded)}
+			>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)